src/lib/file-uploader.js

```diff
@@ -180,7 +180,8 @@ const soundUpload = function (fileData, fileType, soundName, storage, handleSoun
     handleSound(vmSound);
 };
 
-const spriteUpload = function (fileData, fileType, spriteName, storage, handleSprite) {
+const spriteUpload = function (fileData, fileType, spriteName, storage, handleSprite, costumeSuffix) {
```

Comment:
Maybe it would be helpful if there was a jsdoc? I'm not sure what to suggest, this is a lot of arguments.